LoveMachine.Core/PluginInitializer.cs

```diff
@@ -322,6 +343,7 @@ private void DeviceListDrawer(ConfigEntryBase entry)
                     GUILayout.Label("Device Name", GUILayout.Width(columnWidth));
                     GUILayout.Label("Stroker", GUILayout.Width(columnWidth));
                     GUILayout.Label("Vibrators", GUILayout.Width(columnWidth));
+                    GUILayout.Label("Rotators", GUILayout.Width(columnWidth));

```

Comment:
I just noticed this now, but since you're adding a new column, `columns` should be increased to 7.
Sorry, IMGUI doesn't have flexible layout capabilities, it's done manually.